server/app/views/dev/IconsView.java

```diff
@@ -0,0 +1,51 @@
+package views.dev;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.each;
+import static j2html.TagCreator.p;
+import static j2html.TagCreator.table;
+import static j2html.TagCreator.td;
+import static j2html.TagCreator.th;
+import static j2html.TagCreator.tr;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import j2html.tags.ContainerTag;
+import j2html.tags.Tag;
+import play.twirl.api.Content;
+import views.BaseHtmlLayout;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.components.Icons;
+
+public final class IconsView extends BaseHtmlView {
```

Comment:
suggestion (blocking): please document this class

Reply:
Done.